operators/deployment-validation-operator/0.3.0/manifests/deploymentvalidationoperator.0.3.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,158 @@
+                          operator: In
+                          values:
+                          - deployment-validation-operator
+                      topologyKey: kubernetes.io/hostname
+                    weight: 100
+              containers:
+              - env:
+                - name: WATCH_NAMESPACE
+                  value: ""
+                - name: OPERATOR_NAME
+                  value: deployment-validation-operator
+                - name: NAMESPACE_IGNORE_PATTERN
+                  value: "openshift.+|kube-.+"
+                - name: POD_NAME
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.name
+                image: quay.io/deployment-validation-operator/dv-operator:0.3.0
+                imagePullPolicy: Always
+                name: deployment-validation-operator
+                args:
+                - --config /config/deployment-validation-operator-config.yaml
+                resources:
+                  requests:
+                    memory: "256Mi"
+                    cpu: "100n"
+                  limits:
+                    memory: "2Gi"
+                    cpu: "1"
+                volumeMounts:
+                - name: dvo-config
+                  mountPath: /config
+                securityContext:
+                  readOnlyRootFilesystem: true
+              restartPolicy: Always
+              serviceAccountName: deployment-validation-operator
+              terminationGracePeriodSeconds: 30
+              volumes:
+                - name: dvo-config
+                  configMap:
+                    optional: true
+                    name: deployment-validation-operator-config
+      permissions:
+      - rules:
+        - apiGroups:
+          - ""
+          resources:
+          - configmaps
+          - services
+          verbs:
+          - get
+          - create
+          - list
+          - delete
+          - update
+          - watch
+          - patch
+        - apiGroups:
+          - monitoring.coreos.com
+          resources:
+          - servicemonitors
+          verbs:
+          - '*'
+        serviceAccountName: deployment-validation-operator
+    strategy: deployment
+  installModes:
+  - supported: true
+    type: OwnNamespace
+  - supported: true
+    type: SingleNamespace
+  - supported: true
+    type: AllNamespaces
+  - supported: false
+    type: MultiNamespace
+  keywords:
+  - dvo
+  labels:
+    alm-owner-dvo: deployment-validation-operator
+    operated-by: deployment-validation-operator
+  links:
+  - name: repository
+    url: https://github.com/app-sre/deployment-validation-operator
+  - name: containerImage
+    url: https://quay.io/deployment-validation-operator/dv-operator:0.3.0
+  maturity: alpha
+  provider:
+    name: Red Hat
+  selector:
+    matchLabels:
+      alm-owner-dvo: deployment-validation-operator
+      operated-by: deployment-validation-operator
+  version: 0.3.0
```

Comment:
Can you add `replaces: deployment-validation-operator.v0.2.2`

Reply:
`      replaces: deployment-validation-operator.v0.2.2` should be in `spec` level. You can put it after `version: 0.3.0`

Reply:
@npecka 
`replaces` should be as `spec.replaces`, the same level as `spec.version`
```
spec:
  ...
  replaces: deployment-validation-operator.v0.2.2
  version: 0.3.0
```